site/src/pages/TemplateSettingsPage/TemplateSchedulePage/TemplateScheduleAutostart.tsx

```diff
@@ -44,11 +44,13 @@ export const TemplateScheduleAutostart: FC<TemplateScheduleAutostartProps> = ({
 					}[]
 				).map((day) => (
 					<Button
-						fullWidth
+						variant="outline"
+						className="w-full rounded-none"
 						key={day.key}
-						css={{ borderRadius: 0 }}
 						// TODO: Adding a background color would also help
-						color={value.includes(day.value) ? "primary" : "secondary"}
+						style={{
+							color: value.includes(day.value) ? "primary" : "secondary",
+						}}
```

Comment:
these aren't color names that css will know about. I think you probably meant to turn these into classes?